Documentation/platforms/arm/nrf53/boards/thingy53/index.rst

```diff
@@ -78,6 +78,18 @@ nsh_cpunet
 Basic NuttShell configuration for the network core (console enabled in UART0,
 exposed via J-Link VCOM1, at 115200 bps).
 
+rpmsghci_nimble_cpuapp
+----------------------
+
+This configuration enables RPMSG Bluetooth HCI client on the application core
+and uses nimBLE for the host-layer
+
```

Comment:
How does user run and test it?

Reply:
Thank you @raiden00pl, I was referring to how to test the rpmsg example, not how to flash the firmware in the board. Is that explained already in other page?

Reply:
There's not much to explain here. You just flash both images, run the nimble application on the app core and that's it.

Reply:
Ok, thank you @raiden00pl My point was: if someone build this board profile they will know how to do the test or is it something that will generate some request at the mailing list?